packages/graph/hash_graph/lib/graph/src/store/postgres/query/statement/select.rs

```diff
@@ -490,9 +495,10 @@ mod tests {
             SELECT *
             FROM "entities" AS "entities_0_0_0"
             WHERE "entities_0_0_0"."transaction_time" @> $1::TIMESTAMPTZ
+              AND "entities_0_0_0"."decision_time" && $2
               AND "entities_0_0_0"."decision_time" @> now()::TIMESTAMPTZ
             "#,
-            &[&kernel],
+            &[&kernel, &time_projection.image()],
```

Comment:
`"latest"` is sill specified, so `"decision_time"` appears twice here. This will change as soon as we remove the `"latest"` parameter.